topologies/kne/nokia_srlinux_lag.textproto

```diff
@@ -4,7 +4,7 @@ nodes: {
     vendor: NOKIA
     model: "ixr10"
     config: {
-        image: "ghcr.io/nokia/srlinux:latest"
+        image: "srlinux:latest"
```

Comment:
We are just using the kne node defaults here: https://github.com/openconfig/kne/blob/ba4c4208d28fc003ff5faa074e23e8aa24e9ae62/topo/node/srl/srl.go#L322

Please revert this change and update the proto manually on your end or load the intended image into your cluster as `ghcr.io/nokia/srlinux:latest`